plugin/core/views.py

```diff
@@ -406,8 +406,13 @@ def minihtml(view: sublime.View, content: Union[str, Dict[str, str], list], allo
     if is_plain_text:
         return text2html(result)
     else:
-        frontmatter_config = mdpopups.format_frontmatter({'allow_code_wrap': True})
-        return mdpopups.md2html(view, frontmatter_config + result)
+        frontmatter = {
+            "allow_code_wrap": False,
+            "markdown_extensions": [
+                {"pymdownx.escapeall": {}}
```

Comment:
It looks like it won't do "nothing" right now as the value of `allow_code_wrap` has been changed from True to False.
Why is that? That will likely end up working worse.

Also it looks like `escapeall` will replace spaces with `&nbsp;` so that will also make blocks not have possibility to wrap on spaces.

Reply:
Ah yes, allow_code_wrap must remain unchanged. I’m making a lot of mistakes lately 🤔

Reply:
I think  e330a6b fixes all your points.

Reply:
Hopefully. Have not tested but let's try it.

(Also I've realized that this will only process characters escaped with `\` so not sure what's the expected value for `hardbreak` and `nbsp` settings. I guess we should match commonmark if that's what servers assume typically.)